### PR TITLE
Various little changes to get settled into the process

### PR DESCRIPTION
**Pull request by unijohn**
Updated some UG information, replaced brob's placeholder with the placeholders used on advisory board

### CODE REVIEW

File: source/user-groups.html.twig [hunk 1]
```diff
@@ -242,22 +242,22 @@ title: "User Groups"
                     <h3>Inactive or Unknown Status:</h3>
                     <ul>
                         <li>
-                            <a href="http://www.memphisjug.org" target="_blank">Mid-South/Memphis Java User Group</a>
```
Comment (svpernova09) [inline source/user-groups.html.twig]: We should probably keep these URLs around but not link to them?

Comment (svpernova09) [reply source/user-groups.html.twig]: Yeah, I think we should keep the URL and just not link to it.

Comment (unijohn) [reply source/user-groups.html.twig]: e.g. `Mid-South/Memphis Java User Group - http://www.memphisjug.org/` ?

Comment (svpernova09) [reply source/user-groups.html.twig]: Yah: `Mid-South/Memphis Java User Group - http://www.memphisjug.org (Site no longer active)`

**Comment by svpernova09** [comment]
Thanks for the work here! Much appreciated

**Pull request merged**